packages/help-center/src/components/help-center-search-results.tsx

```diff
@@ -224,6 +229,19 @@ function HelpSearchResults( {
 			return <Icon icon={ pageIcon } />;
 		};
 
+		const DeveloperBadge = () => {
+			return (
+				<div className="help-center-search-results__content">
+					{ isResultFromDeveloperWordpress( result.link ) && (
+						<span className="help-center-search-results__badge">
+							{ __( 'DEV', __i18n_text_domain__ ) }
```

Comment:
Slight nit/question: might translator context be needed here as the uppercase could be a bit off-putting? And if so, would it maybe be better to have it sentence case and then use `text-transform: uppercase` instead?

Reply:
I agree 👍 will use `text-transform: uppercase` instead. I think it is more like an acronym than an abbreviation. 
Thank you